src/com/fediroryshchuk/homework7/threads/synchronize/barrier/MyThread.java

```diff
@@ -0,0 +1,27 @@
+package com.fediroryshchuk.homework7.threads.synchronize.barrier;
+
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CyclicBarrier;
+
+public class MyThread implements Runnable {
+    private CyclicBarrier barrier;
+    private String name;
+
+    MyThread(CyclicBarrier barrier, String name) {
+        this.barrier = barrier;
+        this.name = name;
+        new Thread(this).start();
+    }
+
+    public void run() {
+        System.out.println(name);
+
+        try {
+            barrier.await();
+        } catch (BrokenBarrierException e) {
```

Comment:
Since exception handling is the same it could be rewritten
`catch (BrokenBarrierException|InterruptedException e) {`